configuration/scripts/ice_in

```diff
@@ -2,6 +2,9 @@
     days_per_year  = 365
     use_leap_years = .false.
     year_init      = 1997
```

Comment:
shouldn't year_init match the new forcing data?  maybe it doesn't matter, as long as it's properly set in the -s options.  I see below that fyear_init=2005, but I thought that one wasn't used now.  It's a little confusing, would be better to be consistent.

Reply:
I saw this and wondered the same things.  I was reluctant to make this change as I didn't understand the importance or history.  The other thing is that it will change all the gx3 results as well in our regression test.  But I agree 100% that it would be nice if all of this were consistent.  Are there other defaults that need to change.  Should use_leap_years be always true, for instance?  I can make this change as part of this PR, but it will change answers.